engine/settings/base.py

```diff
@@ -42,7 +42,7 @@
 MIRAGE_CIPHER_MODE = "CBC"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = getenv_boolean("DEBUG", default=False)
```

Comment:
nit: things work on my machine without this change. [settings/dev.py](https://github.com/grafana/oncall/blob/6511e439c56b5c3d9f3d82811bd04abe57ca1da5/engine/settings/dev.py#L9)